child_compassion/static/src/js/child_description_wizard.js

```diff
@@ -0,0 +1,53 @@
+openerp.child_compassion = function (instance) {
+    instance.web.child_compassion = instance.web.child_compassion || {};
+    
+    instance.web.views.add('auto_description_form', 'instance.web.child_compassion.AutoDescriptionFormView');
+    instance.web.child_compassion.AutoDescriptionFormView = instance.web.FormView.extend({
+        start: function () {
+            this._super();
+            console.log("Executed the view");
+            this.on('field_changed:child_property_value_ids', this, this.values_changed);
+            this.on('field_changed:project_property_value_ids', this, this.values_changed);
+        },
+        update_value: function(lang, value, value_en) {
+            var value_en_id = '[id="'+value_en[index]+'"]';
+
+            if (value[2].hasOwnProperty('value_'+lang)) {
+                var value_lang = value[2]['value_'+lang];
+                if (value_lang) {
+                    this.$('.oe_desc_'+lang+' iframe').contents().find(value_en_id).text(value_lang);
+                    this.$('.oe_desc_'+lang+' iframe').contents().find(value_en_id).css('color', 'blue');
+                }
+                else {
+                    this.$('.oe_desc_'+lang+' iframe').contents().find(value_en_id).text(value_en[index]);
+                    this.$('.oe_desc_'+lang+' iframe').contents().find(value_en_id).css('color', 'red');
+                }
+            }
+        },
+        values_changed: function() {
+            var value_en = [];
+            $.each(this.$el.find('td[data-field="value_en"]'), function(index, value) {
+                    value_en[index] = value.innerHTML;
+            });
+            if (this.fields.hasOwnProperty('child_property_value_ids')) {
+                var values = this.get_field_value('child_property_value_ids');

```

Comment:
Declare var values outside of if block, otherwise it could be called when never initialized on line 40.